plugins/woocommerce-blocks/assets/js/data/checkout/reducers.ts

```diff
@@ -98,6 +98,15 @@ const reducer = ( state = defaultState, action: CheckoutAction ) => {
 			}
 			break;
 
+		case types.SET_CUSTOMER_PASSWORD:
+			if ( action.customerPassword !== undefined ) {
```

Comment:
I prefer this, it's a little safer
```suggestion
			if ( typeof action.customerPassword !== 'undefined' ) {
```